milestone3/backend/index.js

```diff
@@ -0,0 +1,88 @@
+const express = require("express");
+const app = express();
+app.use(express.json());
+const mongoose = require("mongoose");
+const connection_url =
+  "mongodb+srv://rizenson:h4ipassword@cluster0.2ul0y.mongodb.net/RecipesDB?retryWrites=true&w=majority";
+
+mongoose
+  .connect(connection_url)
+  .then(() => console.log("successful connection"))
+  .catch((error) => console.error("connection unsuccessful"));
+
+app.get("/", (req, res) => {
+  res.send("Hello world!");
+});
+
+const RecipeSchema = new mongoose.Schema(
```

Comment:
not a big deal, but for the sake of organization (and potentially errors), it would be best to put your schema above any requests